pages: add "how to angular" document (#1133)
cloudflare/cloudflare-docs: products/pages/src/content/how-to/deploy-an-angular-application.md

# Deploy an Angular application

> React is an incredibly popular framework for building reactive and powerful frontend applications, built by the open-source team at Facebook.
> 
> In this guide, you'll create a new React application and deploy it using Cloudflare Pages. We'll be using `create-react-app`, a batteries-included tool for generating new React applications.

Reviewer: ```suggestion
Angular is an incredibly popular framework for building reactive and powerful frontend applications.

In this guide, you'll create a new Angular application and deploy it using Cloudflare Pages. We'll be using Angular CLI, a batteries-included tool for generating new Angular applications.
```

Author: Lol shoot, so much for my mental note about coming back to this... thanks!

$ cd my-angular-app
```

With both approaches, the Angular CLI will prompt you for customization choices. Your answers _will not_ affect the rest of this tutorial, so please feel free to select what's ideal for your project.

## Creating a GitHub repository

Create a new GitHub repository by visiting [repo.new](https://repo.new). You can then push your new application to GitHub:

```sh
$ git remote add origin https://github.com/yourgithubusername/githubrepo
$ git branch -M main
$ git push -u origin main
```

## Deploying with Cloudflare Pages

You can deploy your site to Pages by going to the [Cloudflare dashboard](https://dash.cloudflare.com/) and clicking the "Create a project" button.

You will be asked to authorize access to your GitHub account, if you haven't already done so. Cloudflare needs this so that it can monitor and deploy your projects from source. You may narrow access to specific repositories if you prefer this approach; however, you will have to update this list as you attach more repositories to Cloudflare Pages.

Select the new GitHub repository that you created above and, in the configuration section, provide the following information:

<TableLayout>

> | Configuration option | Value           |

Reviewer 2: not a huge deal but these could be lined up

Author: Ok I'll fix this quickly. It doesn't matter to markdown itself but it will for the next person to read this file 😆
